go/packages/dev-mode/agent/forwarder.go

```diff
@@ -0,0 +1,110 @@
+package agent
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+)
+
+type PlatformStartRecord struct {
+	RequestId string `json:"requestId"`
+	Version   string `json:"version"`
+}
+
+type LogMessage struct {
+	Message    string `json:"message"`
+	Timestamp  int64  `json:"timestamp"`
+	SequenceId string `json:"sequenceId"`
+	AccountId  string `json:"accountId"`
+	RequestId  string `json:"requestId"`
+}
+
+type LogPayload struct {
+	RequestId string       `json:"requestId"`
+	TraceId   string       `json:"traceId"`
+	OrgUid    string       `json:"orgUid"`
+	Name      string       `json:"name"`
+	Messages  []LogMessage `json:"messages"`
+}
+
+type LogItem struct {
+	Time    string      `json:"time"`
+	LogType string      `json:"type"`
+	Record  interface{} `json:"record"`
+}
+
+func FindRequestId(logs []LogItem) string {
+	var requestId string = ""
+	for _, log := range logs {
+		if log.LogType == "platform.start" {
+			record := log.Record.(map[string]interface{})
+			requestId = record["requestId"].(string)
+			break
+		}
+	}
+	return requestId
+}
+
+func FormatLogs(logs []LogItem, requestId string) LogPayload {
+	var messages []LogMessage = make([]LogMessage, 0)
```

Comment:
This is really just a style thing, but in general it is more idiomatic when you are both defining and instantiating a variable to write it as  
```suggestion
 messages := make([]LogMessage, 0)
```